mailpoet/views/segments/translations.html

```diff
@@ -135,8 +135,9 @@
 
   'before': _x('before', 'Meaning: "Subscriber subscribed before April"'),
   'after': _x('after', 'Meaning: "Subscriber subscribed after April'),
+  'onOrBefore': __('on or before'),
   'on': _x('on', 'Meaning: "Subscriber subscribed on a given date"'),
-  'notOn': _x('not on', 'Meaning: "Subscriber subscribed on a date other than the given date"'),
+  'onOrAfter': __('on or after'),  'notOn': _x('not on', 'Meaning: "Subscriber subscribed on a date other than the given date"'),
```

Comment:
Just a detail, but we are missing a line break here.

Reply:
Thanks for catching this!